Speak/default.jx3dat

```diff
@@ -14,6 +14,70 @@ local FORCE_TYPE = {
   CHANG_GE  = 22, -- 长歌
   BA_DAO    = 23, -- 霸刀
 }
+
+
 data = {
-  [FORCE_TYPE.WAN_HUA] = { 101--[[握针]], 137--[[局针]], 138--[[提针]], 142--[[长针]], 141--[[毫针]], 14664--[[彼针]], 139--[[锋针]], 2654--[[利针]], 143--[[大针]] },
+  [FORCE_TYPE.SHAO_LIN] = {
+  	[200] = { "hit" }, --[[般若诀]]
+  	[258] = { "hit" }, --[[舍身决]]	
+  },
+  [FORCE_TYPE.WAN_HUA] = { 
+  	[139] = { "hit", "casting" }, --[[锋针]]
+  	[132] = { "hit" }, --[[春泥护花]]
+  	[131] = { "hit" }, --[[碧水滔天]]
+  	[130] = { "hit" }, --[[清新净气]]
+  	[2663] = { "hit" }, --[[听风吹雪]]
+	[14963] = { "hit" }, --[[折叶笼花]]
+	[14965] = { "hit" }, --[[南风吐月]]
+  },
+  [FORCE_TYPE.TIAN_CE] = {
+	[404] = { "hit" }, --[[撼如雷]]
+	[413] = { "hit" }, --[[守如山]]
+	[422] = { "hit" }, --[[啸如虎]]
+	[2628] = { "hit" }, --[[渊]]
+	[15115] = { "hit" }, --[[号令三军]]
+  },
+  [FORCE_TYPE.CHUN_YANG] = {
+	[371] = { "hit" }, --[[镇山河]]
+	[372] = { "hit" }, --[[转乾坤]]
+	[310] = { "hit", "got" }, --[[剑飞惊天]]
```

Comment:
Notice here only CHUN_YANG would speak when "got" 310, but I think you might want every force to take action when "got" 310.
So remove this or consider have a new FORCE_TYPE like -1 that contains all predefine skills that every one would have.